demos/storybook/stories/drawer/with-different-variants.tsx

```diff
@@ -9,7 +9,9 @@ import { Menu } from '@material-ui/icons';
 export const withDifferentVariants = (context: DrawerStoryContext): StoryFnReactReturnType => (
     <Drawer
         open={boolean('open', true)}
-        variant={select('variant', ['permanent', 'persistent', 'temporary'], 'permanent')}
+        variant={select('variant', ['permanent', 'persistent', 'temporary', 'rail'], 'permanent')}
+        condensed={boolean('condensed (rail only)', false)}
+        divider={boolean('divider', false)}
```

Comment:
why `divider` in a story about variants?

Reply:
Jeff specifically asked for this to match the Angular one

Reply:
What was the rationale behind the Angular one to add a divider field? @jeffvg

Reply:
I just remember when we decided to add it for the Angular rail and to simply match up what we can offer between the two frameworks.